### PR TITLE
Set time values to proper precision on decoding

### PR DESCRIPTION
**Pull request by LostKobrakai**

### CODE REVIEW

File: lib/avro_ex/decode.ex [hunk 1]
```diff
@@ -79,7 +79,7 @@ defmodule AvroEx.Decode do
     {val, rest} = variable_integer_decode(data, 0, 0, 64)
     microseconds = zigzag_decode(val)
 
-    {:ok, midnight} = Time.new(0, 0, 0)
+    {:ok, midnight} = Time.new(0, 0, 0, {0, 6})
```
Comment (davydog187) [inline lib/avro_ex/decode.ex:82]: Is this verifiable in unit tests?

Comment (LostKobrakai) [reply lib/avro_ex/decode.ex:82]: These are to fix existing tests, which were failing on my end.

Comment (LostKobrakai) [reply lib/avro_ex/decode.ex:82]: https://github.com/elixir-lang/elixir/issues/12303 let's see if that behaviour is a bug or intended.

Comment (LostKobrakai) [reply lib/avro_ex/decode.ex:88]: Seems like the behaviour for 1.14 was adjusted for the next patch release, but that still leaves 1.14.0 – 1.14.2 with the existing code not working. I've added documentation for the changes, which I feel is the best option here.

Comment (davydog187) [reply lib/avro_ex/decode.ex:88]: https://github.com/elixir-lang/elixir/commit/335c0f6748c4f2ec60cfc8c0577932ebf5d81a4a

It was changed

**Review by davydog187 — APPROVED** [review]

**Comment by davydog187** [comment]
@LostKobrakai merged your PRs! Sorry for the wait I didn't get notified for some reason. Once #88 is merged I'll do a new release

**Pull request merged**